src/UnitConversions.hs

```diff
@@ -4,13 +4,16 @@ import Data.List
 import Data.Ratio
 import Types
 
---unitType: 0 = Metric | 1 = Imperial
-convertRecipeUnits :: Int -> Recipe -> Recipe
+data Conversion = Metric | Imperial | None deriving (Show, Eq)
+
+convertRecipeUnits :: Conversion -> Recipe -> Recipe
 convertRecipeUnits un recp =
-    if un == 0 then
+    if un == Metric then
         recp{ingredients = map convertIngredientToMetric (ingredients recp)}
-    else
+    else if un == Imperial then
```

Comment:
Better to use pattern matching here. Equation is too cumbersome. Something like this will looks better :wink: 
```haskell
case un of
  Metric   -> ...
  Imperial -> ...
  None     -> ...
```